dispatch/static/manager/src/js/components/inputs/DateTimeInput.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { AnchorButton, Popover, Position } from '@blueprintjs/core'
+import { DateTimePicker } from '@blueprintjs/datetime'
+
+import { humanizeDatetime } from '../../util/helpers'
+
+export default function DateTimeInput(props) {
+  const textString = props.value ?
+    humanizeDatetime(props.value)
+    : 'Select a date and time...'
+
+  const picker = (
+    <DateTimePicker
+      value={props.value || new Date()}
+      onChange={props.onChange}
+      timePickerProps={{ // so that state updates when either
+        // date or time are changed
+        onChange: props.onChange
+      }} />
+  )
+
+  return (
+    <div>
+      <Popover
+        content={picker}
+        popoverClassName='pt-popover-content-sizing'
+        position={Position.RIGHT}>
+        <AnchorButton className='c-datetimeinput-button'>
+          <span className='pt-icon-standard pt-icon-calendar' />
+        </AnchorButton>
+      </Popover>
+      <div className='c-datetimeinput-text'>
+        {textString}
+      </div>
+    </div>
+  )
+}
```

Comment:
I'd like to change the design of the date field to make it a bit more condensed. Can you use a [control group](http://blueprintjs.com/docs/#core/components/forms/control-group) to do something like this?

![screen shot 2017-06-02 at 5 05 59 pm](https://cloud.githubusercontent.com/assets/2547035/26746553/f53abe62-47b5-11e7-9da3-39832c177c3d.png)
![screen shot 2017-06-02 at 5 05 49 pm](https://cloud.githubusercontent.com/assets/2547035/26746554/f56e8d14-47b5-11e7-9274-ab3c68bd92b4.png)

Clicking on the calendar icon or on the field directly would open up the popover for editing